frontend/src/v5/ui/components/shared/breadcrumbs/breadcrumbs.component.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Container, Breadcrumb, InteractiveBreadcrumb } from './breadcrumbs.styles';
+
+export const Breadcrumbs = (): JSX.Element => {
+	const getBreadcrumbs = ['ABCDEFGHIJKLMNOPQRSTUVWXYZ', 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'];
```

Comment:
Reviewing the code from the basic layout PR I think this can be left in here (in the near future a selector would get this data) but the options for navigation in top navigation component should be populated and passed on here to to topnavigation.